app/scripts/controller-init/seedless-onboarding/key-deriver.ts

```diff
@@ -0,0 +1,28 @@
+import type { ToprfKeyDeriver } from '@metamask/seedless-onboarding-controller';
+
+const TOPRF_KEY_DERIVATION_ITERATIONS = 600_000;
```

Comment:
need to adjust this number so that performance is satisfactory across all platforms.
(if we inject a toprf key deriver, it needs to do exactly the same on all platforms.)